Dockerfile

```diff
@@ -16,3 +16,6 @@ ADD https://github.com/FFIG/ffig/archive/master.zip /home/ffig/ffig
 
 # Copy in the content of this repository
 COPY . /home/ffig/rest-api
+WORKDIR /home/ffig/flask
+EXPOSE 5000
+RUN python ./run.py
```

Comment:
Apologies for any confusion - I was happy with the `WORKDIR` and `EXPOSE` lines; it was only the `RUN` line I wanted to ask about.